chamber/utils/datastructures.py

```diff
@@ -133,6 +133,11 @@ def __init__(self, categories):
     def get_allowed_states(self, category):
         return self.categories.get(category, ())
 
+    def get_category(self, key):
+        for category, items in self.categories.items():
+            if key in items:
+                return category
```

Comment:
tady chybí return mimo ten for cyklus

Reply:
pravda pravda

Reply:
Implicitně fce bez returnu vrací None, ne?

Reply:
explicit is better than implicit :trollface: